src/api/is-device-supported.js

```diff
@@ -0,0 +1,15 @@
+import getDeviceSize from '../utils/get-device-size';
+
+
+/**
+ * Checks if the device is supported as the target for the `resize` API method.
+ * @function
+ * @async
+ * @name isDeviceSupportedAsResizeTarget
+ * @param {string} deviceName - Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.
+ */
```

Comment:
BTW, just figured out that `viewport-list` package fetches viewports from the website each time it's run. I don't think it's quite reliable, also it's render TestCafe unusable in the environments without internet access. Can we include http://viewportsizes.com/devices.json into `browser-natives` and just create list on initialization. With this approach we'll be able to query device sizes synchronously, so device argument can be validated using `Assignable` machinery with early errors.